nerdlets/pathpoint-nerdlet/components/Modal/TouchpointsEditor.js

```diff
@@ -903,17 +929,70 @@ class BodyTouchpointsEditor extends Component {
     });
   };
 
+  TimeRangeTransform(timeRange) {
+    let time_start = 0;
+    let time_end = 0;
+    if (timeRange === '5 MINUTES AGO') {
+      return timeRange;
+    }
+    switch (timeRange) {
+      case '30 MINUTES AGO':
+        time_start = Math.floor(Date.now() / 1000) - 35 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 30 * 60;
+        break;
+      case '60 MINUTES AGO':
+        time_start = Math.floor(Date.now() / 1000) - 65 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 60 * 60;
+        break;
+      case '3 HOURS AGO':
+        time_start = Math.floor(Date.now() / 1000) - 3 * 60 * 60 - 5 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 3 * 60 * 60;
+        break;
+      case '6 HOURS AGO':
+        time_start = Math.floor(Date.now() / 1000) - 6 * 60 * 60 - 5 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 6 * 60 * 60;
+        break;
+      case '12 HOURS AGO':
+        time_start = Math.floor(Date.now() / 1000) - 12 * 60 * 60 - 5 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 12 * 60 * 60;
+        break;
+      case '24 HOURS AGO':
+        time_start = Math.floor(Date.now() / 1000) - 24 * 60 * 60 - 5 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 24 * 60 * 60;
+        break;
+      case '3 DAYS AGO':
+        time_start = Math.floor(Date.now() / 1000) - 3 * 24 * 60 * 60 - 5 * 60;
+        time_end = Math.floor(Date.now() / 1000) - 3 * 24 * 60 * 60;
+        break;
+      case '7 DAYS AGO':
```

Comment:
@portoren I found a bug here.


`7 Days Ago` is using 3 days -- so it is identical to `3 Days Ago`

Reply:
Done